### PR TITLE
Fix node constraints & add label for readme nodetype

### PR DESCRIPTION
**Pull request by Sebobo**

### CODE REVIEW

File: Configuration/NodeTypes.yaml [hunk 1]
```diff
@@ -68,13 +68,23 @@ githubProperty: &githubProperty
...
+#
+# TYPO3.Neos:ContentCollection
 'TYPO3.Neos:ContentCollection':
   constraints:
     nodeTypes:
       'Neos.MarketPlace:Version': FALSE
       'Neos.MarketPlace:Dist': FALSE
       'Neos.MarketPlace:Source': FALSE
       'Neos.MarketPlace:People': FALSE
+      'Neos.MarketPlace:Readme': FALSE
```
Comment (dfeyer) [inline Configuration/NodeTypes.yaml]: Why sometimes TRUE and sometimes true ;) need to be consistent, no ?

**Comment by dfeyer** [comment]
Thanks, just a small nitpick about TRUE / true ;)

**Comment by Sebobo** [comment]
This lowercase/uppercase thing starts to annoy me so I finally read the spec :D

**Comment by Sebobo** [comment]
@dfeyer ping

**Comment by dfeyer** [comment]
Sorry for the delay,

**Pull request merged**